docarray/typing/url/image_url.py

```diff
@@ -33,10 +32,7 @@ def validate(
         url = super().validate(value, field, config)  # basic url validation
         has_image_extension = any(url.endswith(ext) for ext in IMAGE_FILE_FORMATS)
```

Comment:
To align with other modalities, we should also allow no extension at all. So to pass validation the rule should be:
- Either there is an extension, and the extension is in `IMAGE_FILE_FORMATS`
- OR there is not extension at all
You can take a look at how this is handled for audio data, for example.

And please also adjust the error message below accordingly :) Thanks!